scrapper.py

```diff
@@ -25,39 +36,159 @@ class Crawler:
     """
     Crawler implementation
     """
-    def __init__(self, seed_urls, max_articles: int):
-        pass
 
-    def _extract_url(self, article_bs):
-        pass
+    def __init__(self, seed_urls, max_articles: int):
+        self.max_articles = max_articles
+        self.seed_urls = seed_urls
+        self.count_articles = 0
+        self.urls = []
+
+    def _extract_url(self, article_bs):  # I don't understand if I need to collect seed urls (from main to year)
+        # or just put it in scrapper_config.
+        articles_code_stored_here = article_bs.find_all('div', {'class': 'articles'})
+        for article_code in articles_code_stored_here:
+            links_articles = article_code.find_all('a')
+            for link in links_articles:
+                if self.max_articles == self.count_articles:
+                    break
+                try:
+                    art_link = link["href"]
+                    if art_link.startswith('/') and 'pdf' not in art_link:
+                        self.urls.append('https://periodical.pstgu.ru' + link["href"])
+                        self.count_articles += 1
+                except KeyError:
+                    continue
 
     def find_articles(self):
         """
         Finds articles
-        """
-        pass
+        """  # I also don't understand what I need to do here, because this code can be moved to extract_url
+        for seed_url in self.seed_urls:
+            issue_page = requests.get(seed_url)
```

Comment:
any attempts to make the remote server request more human like? do you recall several strategies that wllo Python requests look like human-made?